packages/config/src/options.js

```diff
@@ -281,8 +281,14 @@ export function getNuxtConfig(_options) {
     consola.warn('vendor has been deprecated due to webpack4 optimization')
   }
 
+  // Disable CSS extraction due to incompatibility with thread-loader
+  if (options.build && options.build.parallel && options.build.extractCSS) {
+    options.build.extractCSS = false
+    consola.warn('extractCSS cannot work with parallel loaders due to missing webpack context')
+  }
+
   // build.extractCSS.allChunks has no effect
-  if (typeof options.build.extractCSS.allChunks !== 'undefined') {
+  if (typeof options.build.extractCSS === 'object' && options.build.extractCSS.allChunks) {
```

Comment:
I think you can just revert this part since `extractCSS` is not undefined by `false` after your previous change